recipes/corpus/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "Corpus" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ffa894899350675f874bab9e93ef07326aeb5fc1e9ef62d25729d4b2903c63cc
+
+build:
```

Comment:
Can this be a `noarch:python` package? 
https://conda-forge.org/docs/meta.html#building-noarch-packages